app/controllers/api_controller.rb

```diff
@@ -12,14 +12,17 @@ def initialize
       @records = {}
     end
 
-    def <<(response) # { records: records, associated: records, errors: errors, data: data }
-      if records = response.delete(:records)
+    # { records: records, associated: records, errors: errors, data: data }
+    def <<(response)
```

Comment:
Assignment Branch Condition size for << is too high. [23.96/15]<br>Method has too many lines. [23/10]